gitlab/README.rst

```diff
@@ -0,0 +1,85 @@
+======
+Gitlab
+======
+
+.. 
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! source digest: sha256:5fe540da3a23a490321ee98260fbc37dcc03fb8ceac994d75890e7b2af7753b3
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fserver--backend-lightgray.png?logo=github
+    :target: https://github.com/OCA/server-backend/tree/16.0/gitlab
+    :alt: OCA/server-backend
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/server-backend-16-0/server-backend-16-0-gitlab
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runboat-Try%20me-875A7B.png
+    :target: https://runboat.odoo-community.org/builds?repo=OCA/server-backend&target_branch=16.0
+    :alt: Try me on Runboat
+
+|badge1| |badge2| |badge3| |badge4| |badge5|
+
+This module does not provide a functionality by itself but a base model
+to implement gitlab related features
```

Comment:
this looks very generic, and quite hard to understand the perimeter of this module.
Could you mention links to extra modules that depends on this module ? 

thanks !

Reply:
@legalsylvain ,the extra module is not yet added as its dependent on this module.The module is aimed at recording user activities like commits, pull requests (merge requests), opening issues etc.

Reply:
Hi maybe we could share concept and code with https://github.com/OCA/interface-github module, dont you think ? 
With some refactor it looks me possible at a first sight

Reply:
Here is the link to the extra module that depends on this module - https://github.com/OCA/vertical-association/pull/178. Actually, its not identical to the interface-github